### ISSUE TITLE
Every project property page should have corresponding preference page.

### ISSUE THREAD

**Issue by paulvi** [angelozerr/tern.java#316]
For example if there is page for "ECMAScript compliance + JSDoc" in project properties, there should be the same page in workspace preferences to set the defaults for workspace.

![](https://cloud.githubusercontent.com/assets/1932211/9664034/c0a9f428-5267-11e5-9eb6-92d17befd130.png)

**Comment by angelozerr** [comment]
Yes I have though about that, but the problem is that Global Preferences update classic Eclipse preferences and Project Properties update the .tern-project. If I find time, I will try to do that.
